### ISSUE TITLE
[Accessibility][Programmatic access]: Screen reader doesn't give any feedback while adding facepiles using 'Add friend' button. (Bug 8)

### ISSUE THREAD

**Issue by jdhuntington** [microsoft/fluentui#10862]
Tracking all of the issues brought up with the 1/4 accessibility pass on Facepile control 
Details on individual bugs can be found on [SharePoint](https://microsoft-my.sharepoint.com/:x:/r/personal/v-amaver_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7Bfd43fcc5-515d-47d8-946d-0d4862a56278%7D&action=default&uid=%7BFD43FCC5-515D-47D8-946D-0D4862A56278%7D&ListItemId=1647&ListId=%7B395E1ECB-5002-440E-81C5-6A090E98077C%7D&odsp=1&env=prodbubble) 

Breakout of issue #7534

**Comment by micahgodbolt** [comment]
@aneeshack4 this sounds like it is part of the facepile narrator revamp in #10851 

Once we know how a facepile is supposed to announce on focus, we can also determine how it should announce when it changes.

**Comment by aneeshack4** [comment]
@Markionium @mtennoe any thoughts on how Facepile is supposed to announce on focus and changes? 
@micahgodbolt did some investigation into Facepile's DOM structure and it looks like `listbox` isn't being used properly here/per w3 spec: 
![image](https://user-images.githubusercontent.com/4161791/70757615-1732ec80-1cf5-11ea-92fa-8cf26ecb35a8.png)
`listbox` should technically be used to identify an element that creates a list from which a user may select one or more static items. Not sure if that fits the Facepile situation... is it really a `listbox` where each item should be actionable or should the role be provided by user?

**Comment by mtennoe** [comment]
I'll let @Markionium comment on that as he has the best accessibility overview :)

**Comment by aneeshack4** [comment]
The team that owns Facepile is tracking this separately and will get a fix for it out soon. For now I'll assign this to @mtennoe and @Markionium so they can update these issues when those fixes are done.
